Repository/Models/FollowingMovie.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+
+#nullable disable
+
+namespace Repository.Models
+{
+    public partial class FollowingMovie
```

Comment:
porpose

Reply:
That is a database model, created by the database-first scaffolding tool.